packages/alea-frontend/pages/api/acl-utils/acl-common-utils.ts

```diff
@@ -3,14 +3,25 @@ import { NextApiResponse } from 'next';
 import { executeAndEndSet500OnError, getUserIdOrSetError } from '../comment-utils';
 import { CACHE_STORE } from './cache-store';
 
+export async function checkResourcesassociatedOrSet500OnError(
+  aclId: string,
+  res: any 
+): Promise<boolean> {
+  const resourceId = await executeAndEndSet500OnError(
+    'select resourceId from resourceaccess where aclId=?',
+    [aclId],
+    res
+  );
+
+  if (resourceId?.length) return true;
```

Comment:
error handler's behavior is not correct. 

SEe
https://github.com/KWARC/ALeA/blob/main/backend_and_general_guidelines.md#27-returning-api-errors-outside-handler